### ISSUE TITLE
Database Error after update to 7.29

### ISSUE THREAD

**Issue by XenonSup** [flathub/org.signal.Signal#774]
When launching signal after the latest update `76af70909d865cc325ad167c2cc52961491bcd5e290daa902d7e39833ea4796b`, I got the following dialog
>A databse error occured. You can copy the error and contact Signal support to help fix the issue. If you need to use Signal right away you can delete your data and restart.
>
> Contact support by visiting: https://support.signal.org/error
>
>|Copy error and quit|            |Delete data and restart|

"Copy error and quit" doesn't copy anything, so I relaunched Signal from terminal
```
$ flatpak run org.signal.Signal
Debug: Using password store: gnome-libsecret
Debug: Will run signal with the following arguments: --password-store=gnome-libsecret
Debug: Additionally, user gave: 
Set Windows Application User Model ID (AUMID) { AUMID: 'org.whispersystems.signal-desktop' }
NODE_ENV production
NODE_CONFIG_DIR /app/Signal/resources/app.asar/config
NODE_CONFIG {}
ALLOW_CONFIG_MUTATIONS undefined
HOSTNAME xenon
NODE_APP_INSTANCE undefined
SUPPRESS_NO_CONFIG_WARNING undefined
SIGNAL_ENABLE_HTTP undefined
userData: /home/xenon/.var/app/org.signal.Signal/config/Signal
config/get: Successfully read user config file
config/get: Successfully read ephemeral config file
making app single instance
LaunchProcess: failed to execvp:
xdg-settings
LaunchProcess: failed to execvp:
xdg-settings
Gtk-Message: 07:52:31.455: Failed to load module "pk-gtk-module"
Gtk-Message: 07:52:31.457: Failed to load module "pk-gtk-module"
{"level":30,"time":"2024-10-17T11:52:31.872Z","msg":"got fast localeOverride setting null"}
{"level":30,"time":"2024-10-17T11:52:31.873Z","msg":"app.ready: hour cycle preference: UnknownPreference"}
{"level":30,"time":"2024-10-17T11:52:31.873Z","msg":"app.ready: preferred system locales: en-US, en"}
{"level":30,"time":"2024-10-17T11:52:31.874Z","msg":"locale: Supported locales: af-ZA, ar, az-AZ, bg-BG, bn-BD, bs-BA, ca, cs, da, de, el, en, es, et-EE, eu, fa-IR, fi, fr, ga-IE, gl-ES, gu-IN, he, hi-IN, hr-HR, hu, id, it, ja, ka-GE, kk-KZ, km-KH, kn-IN, ko, ky-KG, lt-LT, lv-LV, mk-MK, ml-IN, mr-IN, ms, my, nb, nl, pa-IN, pl, pt-BR, pt-PT, ro-RO, ru, sk-SK, sl-SI, sq-AL, sr, sv, sw, ta-IN, te-IN, th, tl-PH, tr, ug, uk-UA, ur, vi, yue, zh-CN, zh-HK, zh-Hant"}
{"level":30,"time":"2024-10-17T11:52:31.874Z","msg":"locale: Preferred locales: en-US, en"}
{"level":30,"time":"2024-10-17T11:52:31.874Z","msg":"locale: Locale Override: null"}
{"level":30,"time":"2024-10-17T11:52:31.878Z","msg":"locale: Matched locale: en"}
{"level":40,"time":"2024-10-17T11:52:31.941Z","msg":"intl.onWarn [@formatjs/intl] \"defaultRichTextElements\" was specified but \"message\" was not pre-compiled. \nPlease consider using \"@formatjs/cli\" to pre-compile your messages for performance.\nFor more details see https://formatjs.io/docs/getting-started/message-distribution"}
{"level":30,"time":"2024-10-17T11:52:31.941Z","msg":"locale: Text info direction for en: ltr"}
{"level":30,"time":"2024-10-17T11:52:31.942Z","msg":"getSQLKey: decrypting key"}
{"level":30,"time":"2024-10-17T11:52:31.943Z","msg":"getSystemTraySetting got value MinimizeToSystemTray"}
{"level":30,"time":"2024-10-17T11:52:31.943Z","msg":"getSystemTraySetting returning MinimizeToSystemTray"}
{"level":30,"time":"2024-10-17T11:52:31.945Z","msg":"app ready"}
{"level":30,"time":"2024-10-17T11:52:31.945Z","msg":"starting version 7.29.0"}
{"level":30,"time":"2024-10-17T11:52:31.946Z","msg":"media access status [object Undefined] [object Undefined]"}
{"level":30,"time":"2024-10-17T11:52:31.949Z","msg":"got fast theme-setting value system"}
{"level":30,"time":"2024-10-17T11:52:31.962Z","msg":"got fast theme-setting value system"}
{"level":30,"time":"2024-10-17T11:52:31.963Z","msg":"got fast spellcheck setting true"}
{"level":30,"time":"2024-10-17T11:52:31.964Z","msg":"visibleOnAnyScreen(window): x=640, y=32, width=1280, height=1011"}
{"level":30,"time":"2024-10-17T11:52:31.964Z","msg":"visibleOnAnyScreen(display #0): x=0, y=0, width=1920, height=1080"}
{"level":30,"time":"2024-10-17T11:52:31.965Z","msg":"Initializing BrowserWindow config: {\"show\":false,\"width\":1280,\"height\":1011,\"minWidth\":300,\"minHeight\":200,\"autoHideMenuBar\":true,\"titleBarStyle\":\"default\",\"backgroundColor\":\"#3a76f0\",\"webPreferences\":{\"devTools\":false,\"spellcheck\":true,\"enableBlinkFeatures\":\"CSSPseudoDir,CSSLogical\",\"enablePreferredSizeMode\":true,\"nodeIntegration\":false,\"nodeIntegrationInWorker\":false,\"sandbox\":false,\"contextIsolation\":true,\"preload\":\"[REDACTED]/preload.wrapper.js\",\"backgroundThrottling\":true,\"disableBlinkFeatures\":\"Accelerated2dCanvas,AcceleratedSmallCanvases\"},\"icon\":\"[REDACTED]/images/signal-logo-desktop-linux.png\",\"x\":640,\"y\":32}"}
{"level":30,"time":"2024-10-17T11:52:32.033Z","msg":"spellcheck: user locales: [\"en-US\",\"en\"]"}
{"level":30,"time":"2024-10-17T11:52:32.034Z","msg":"spellcheck: available spellchecker languages: [\"af\",\"bg\",\"ca\",\"cs\",\"cy\",\"da\",\"de\",\"de-DE\",\"el\",\"en\",\"en-AU\",\"en-CA\",\"en-GB\",\"en-GB-oxendict\",\"en-US\",\"es\",\"es-419\",\"es-AR\",\"es-ES\",\"es-MX\",\"es-US\",\"et\",\"fa\",\"fo\",\"fr\",\"fr-FR\",\"he\",\"hi\",\"hr\",\"hu\",\"hy\",\"id\",\"it\",\"it-IT\",\"ko\",\"lt\",\"lv\",\"nb\",\"nl\",\"pl\",\"pt\",\"pt-BR\",\"pt-PT\",\"ro\",\"ru\",\"sh\",\"sk\",\"sl\",\"sq\",\"sr\",\"sv\",\"ta\",\"tg\",\"tr\",\"uk\",\"vi\"]"}
{"level":30,"time":"2024-10-17T11:52:32.034Z","msg":"spellcheck: setting languages to: [\"en-US\",\"en\"]"}
2024-10-17 07:52:32.287: ERROR CORE sqlcipher_page_cipher: hmac check failed for pgno=1
2024-10-17 07:52:32.287: ERROR CORE sqlite3Codec: error decrypting page 1 data: 1
2024-10-17 07:52:32.287: ERROR CORE sqlcipher_codec_ctx_set_error 1
{"level":40,"time":"2024-10-17T11:52:32.287Z","msg":"MainSQL: Database log code=26: file is not a database in \"PRAGMA journal_mode = WAL\""}
{"level":30,"time":"2024-10-17T11:52:32.287Z","msg":"MainSQL: migrateDatabase: Migration without cipher change failed"}
2024-10-17 07:52:32.352: ERROR CORE sqlcipher_page_cipher: hmac check failed for pgno=1
2024-10-17 07:52:32.352: ERROR CORE sqlite3Codec: error decrypting page 1 data: 1
2024-10-17 07:52:32.352: ERROR CORE sqlcipher_codec_ctx_set_error 1
{"level":40,"time":"2024-10-17T11:52:32.352Z","msg":"MainSQL: Database log code=26: statement aborts at 2: [PRAGMA user_version] file is not a database"}
{"level":50,"time":"2024-10-17T11:52:32.353Z","msg":"MainSQL: Database startup error: SqliteError: file is not a database\n    at Database.pragma ([REDACTED]/node_modules/@signalapp/better-sqlite3/lib/methods/pragma.js:11:31)\n    at getUserVersion ([REDACTED]/ts/sql/util.js:132:13)\n    at migrateSchemaVersion ([REDACTED]/ts/sql/Server.js:409:54)\n    at openAndMigrateDatabase ([REDACTED]/ts/sql/Server.js:441:5)\n    at openAndSetUpSQLCipher ([REDACTED]/ts/sql/Server.js:463:14)\n    at initialize ([REDACTED]/ts/sql/Server.js:510:10)\n    at MessagePort.<anonymous> ([REDACTED]/ts/sql/mainWorker.js:69:41)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:820:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
{"level":50,"time":"2024-10-17T11:52:32.353Z","msg":"Failed to get zoom factor {\"name\":\"SqliteError\"}"}
{"level":30,"time":"2024-10-17T11:52:32.752Z","msg":"got fast theme-setting value system"}
{"level":50,"time":"2024-10-17T11:52:33.427Z","msg":"sql.initialize was unsuccessful; returning early"}
{"level":30,"time":"2024-10-17T11:52:33.428Z","msg":"close event {\"readyForShutdown\":false,\"shouldQuit\":false}"}
{"level":30,"time":"2024-10-17T11:52:33.428Z","msg":"maybeRequestCloseConfirmation: Checking to see if close confirmation is needed"}
{"level":50,"time":"2024-10-17T11:53:02.118Z","msg":"onDatabaseError: Quitting application"}
{"level":30,"time":"2024-10-17T11:53:02.121Z","msg":"main window closed event"}
{"level":30,"time":"2024-10-17T11:53:02.122Z","msg":"quit event {\"hasEventBeenPrevented\":false,\"windowCount\":0,\"mainWindowExists\":false}"}
{"level":50,"time":"2024-10-17T11:53:02.127Z","msg":"Error occurred in handler for 'sql-channel:read': {\"name\":\"SqliteError\"}"}
{"level":50,"time":"2024-10-17T11:53:02.129Z","msg":"maybeRequestCloseConfirmation: Response never received; continuing with close."}
{"level":50,"time":"2024-10-17T11:53:02.130Z","msg":"MainSQL close, failed: SqliteError: file is not a database\n    at Database.pragma ([REDACTED]/node_modules/@signalapp/better-sqlite3/lib/methods/pragma.js:11:31)\n    at getUserVersion ([REDACTED]/ts/sql/util.js:132:13)\n    at migrateSchemaVersion ([REDACTED]/ts/sql/Server.js:409:54)\n    at openAndMigrateDatabase ([REDACTED]/ts/sql/Server.js:441:5)\n    at openAndSetUpSQLCipher ([REDACTED]/ts/sql/Server.js:463:14)\n    at initialize ([REDACTED]/ts/sql/Server.js:510:10)\n    at MessagePort.<anonymous> ([REDACTED]/ts/sql/mainWorker.js:69:41)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:820:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
```

I thought this is just an issue with the latest update, so I downgraded and relaunched signal, but issue persists

`$ sudo flatpak update --commit=be19a23652d102b0f87106609b06a7f50972bfae7540a02767a04bf1ae169cd7 org.signal.Signal`
```
$ flatpak run org.signal.Signal
Debug: Using password store: gnome-libsecret
Debug: Will run signal with the following arguments: --password-store=gnome-libsecret
Debug: Additionally, user gave: 
Set Windows Application User Model ID (AUMID) { AUMID: 'org.whispersystems.signal-desktop' }
NODE_ENV production
NODE_CONFIG_DIR /app/Signal/resources/app.asar/config
NODE_CONFIG {}
ALLOW_CONFIG_MUTATIONS undefined
HOSTNAME xenon
NODE_APP_INSTANCE undefined
SUPPRESS_NO_CONFIG_WARNING undefined
SIGNAL_ENABLE_HTTP undefined
userData: /home/xenon/.var/app/org.signal.Signal/config/Signal
config/get: Successfully read user config file
config/get: Successfully read ephemeral config file
making app single instance
LaunchProcess: failed to execvp:
xdg-settings
LaunchProcess: failed to execvp:
xdg-settings
Gtk-Message: 07:59:28.605: Failed to load module "pk-gtk-module"
Gtk-Message: 07:59:28.608: Failed to load module "pk-gtk-module"
{"level":30,"time":"2024-10-17T11:59:29.040Z","msg":"got fast localeOverride setting null"}
{"level":30,"time":"2024-10-17T11:59:29.042Z","msg":"app.ready: hour cycle preference: UnknownPreference"}
{"level":30,"time":"2024-10-17T11:59:29.042Z","msg":"app.ready: preferred system locales: en-US, en"}
{"level":30,"time":"2024-10-17T11:59:29.042Z","msg":"locale: Supported locales: af-ZA, ar, az-AZ, bg-BG, bn-BD, bs-BA, ca, cs, da, de, el, en, es, et-EE, eu, fa-IR, fi, fr, ga-IE, gl-ES, gu-IN, he, hi-IN, hr-HR, hu, id, it, ja, ka-GE, kk-KZ, km-KH, kn-IN, ko, ky-KG, lt-LT, lv-LV, mk-MK, ml-IN, mr-IN, ms, my, nb, nl, pa-IN, pl, pt-BR, pt-PT, ro-RO, ru, sk-SK, sl-SI, sq-AL, sr, sv, sw, ta-IN, te-IN, th, tl-PH, tr, ug, uk-UA, ur, vi, yue, zh-CN, zh-HK, zh-Hant"}
{"level":30,"time":"2024-10-17T11:59:29.042Z","msg":"locale: Preferred locales: en-US, en"}
{"level":30,"time":"2024-10-17T11:59:29.043Z","msg":"locale: Locale Override: null"}
{"level":30,"time":"2024-10-17T11:59:29.046Z","msg":"locale: Matched locale: en"}
{"level":40,"time":"2024-10-17T11:59:29.097Z","msg":"intl.onWarn [@formatjs/intl] \"defaultRichTextElements\" was specified but \"message\" was not pre-compiled. \nPlease consider using \"@formatjs/cli\" to pre-compile your messages for performance.\nFor more details see https://formatjs.io/docs/getting-started/message-distribution"}
{"level":30,"time":"2024-10-17T11:59:29.097Z","msg":"locale: Text info direction for en: ltr"}
{"level":30,"time":"2024-10-17T11:59:29.098Z","msg":"getSQLKey: decrypting key"}
{"level":30,"time":"2024-10-17T11:59:29.098Z","msg":"getSystemTraySetting got value MinimizeToSystemTray"}
{"level":30,"time":"2024-10-17T11:59:29.099Z","msg":"getSystemTraySetting returning MinimizeToSystemTray"}
{"level":30,"time":"2024-10-17T11:59:29.101Z","msg":"app ready"}
{"level":30,"time":"2024-10-17T11:59:29.101Z","msg":"starting version 7.28.0"}
{"level":30,"time":"2024-10-17T11:59:29.101Z","msg":"media access status [object Undefined] [object Undefined]"}
{"level":30,"time":"2024-10-17T11:59:29.105Z","msg":"got fast theme-setting value system"}
{"level":30,"time":"2024-10-17T11:59:29.127Z","msg":"got fast theme-setting value system"}
{"level":30,"time":"2024-10-17T11:59:29.127Z","msg":"got fast spellcheck setting true"}
{"level":30,"time":"2024-10-17T11:59:29.128Z","msg":"visibleOnAnyScreen(window): x=640, y=32, width=1280, height=1011"}
{"level":30,"time":"2024-10-17T11:59:29.128Z","msg":"visibleOnAnyScreen(display #0): x=0, y=0, width=1920, height=1080"}
{"level":30,"time":"2024-10-17T11:59:29.128Z","msg":"Initializing BrowserWindow config: {\"show\":false,\"width\":1280,\"height\":1011,\"minWidth\":300,\"minHeight\":200,\"autoHideMenuBar\":true,\"titleBarStyle\":\"default\",\"backgroundColor\":\"#3a76f0\",\"webPreferences\":{\"devTools\":false,\"spellcheck\":true,\"enableBlinkFeatures\":\"CSSPseudoDir,CSSLogical\",\"enablePreferredSizeMode\":true,\"nodeIntegration\":false,\"nodeIntegrationInWorker\":false,\"sandbox\":false,\"contextIsolation\":true,\"preload\":\"[REDACTED]/preload.wrapper.js\",\"backgroundThrottling\":true,\"disableBlinkFeatures\":\"Accelerated2dCanvas,AcceleratedSmallCanvases\"},\"icon\":\"[REDACTED]/images/signal-logo-desktop-linux.png\",\"x\":640,\"y\":32}"}
{"level":30,"time":"2024-10-17T11:59:29.204Z","msg":"spellcheck: user locales: [\"en-US\",\"en\"]"}
{"level":30,"time":"2024-10-17T11:59:29.204Z","msg":"spellcheck: available spellchecker languages: [\"af\",\"bg\",\"ca\",\"cs\",\"cy\",\"da\",\"de\",\"de-DE\",\"el\",\"en\",\"en-AU\",\"en-CA\",\"en-GB\",\"en-GB-oxendict\",\"en-US\",\"es\",\"es-419\",\"es-AR\",\"es-ES\",\"es-MX\",\"es-US\",\"et\",\"fa\",\"fo\",\"fr\",\"fr-FR\",\"he\",\"hi\",\"hr\",\"hu\",\"hy\",\"id\",\"it\",\"it-IT\",\"ko\",\"lt\",\"lv\",\"nb\",\"nl\",\"pl\",\"pt\",\"pt-BR\",\"pt-PT\",\"ro\",\"ru\",\"sh\",\"sk\",\"sl\",\"sq\",\"sr\",\"sv\",\"ta\",\"tg\",\"tr\",\"uk\",\"vi\"]"}
{"level":30,"time":"2024-10-17T11:59:29.204Z","msg":"spellcheck: setting languages to: [\"en-US\",\"en\"]"}
2024-10-17 07:59:29.427: ERROR CORE sqlcipher_page_cipher: hmac check failed for pgno=1
2024-10-17 07:59:29.427: ERROR CORE sqlite3Codec: error decrypting page 1 data: 1
2024-10-17 07:59:29.427: ERROR CORE sqlcipher_codec_ctx_set_error 1
{"level":40,"time":"2024-10-17T11:59:29.428Z","msg":"MainSQL: Database log code=26: file is not a database in \"PRAGMA journal_mode = WAL\""}
{"level":30,"time":"2024-10-17T11:59:29.428Z","msg":"MainSQL: migrateDatabase: Migration without cipher change failed"}
2024-10-17 07:59:29.491: ERROR CORE sqlcipher_page_cipher: hmac check failed for pgno=1
2024-10-17 07:59:29.491: ERROR CORE sqlite3Codec: error decrypting page 1 data: 1
2024-10-17 07:59:29.491: ERROR CORE sqlcipher_codec_ctx_set_error 1
{"level":40,"time":"2024-10-17T11:59:29.491Z","msg":"MainSQL: Database log code=26: statement aborts at 2: [PRAGMA user_version] file is not a database"}
{"level":50,"time":"2024-10-17T11:59:29.492Z","msg":"MainSQL: Database startup error: SqliteError: file is not a database\n    at Database.pragma ([REDACTED]/node_modules/@signalapp/better-sqlite3/lib/methods/pragma.js:11:31)\n    at getUserVersion ([REDACTED]/ts/sql/util.js:132:13)\n    at migrateSchemaVersion ([REDACTED]/ts/sql/Server.js:406:54)\n    at openAndMigrateDatabase ([REDACTED]/ts/sql/Server.js:438:5)\n    at openAndSetUpSQLCipher ([REDACTED]/ts/sql/Server.js:460:14)\n    at initialize ([REDACTED]/ts/sql/Server.js:507:10)\n    at MessagePort.<anonymous> ([REDACTED]/ts/sql/mainWorker.js:69:41)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:820:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
{"level":50,"time":"2024-10-17T11:59:29.492Z","msg":"Failed to get zoom factor {\"name\":\"SqliteError\"}"}
{"level":30,"time":"2024-10-17T11:59:29.885Z","msg":"got fast theme-setting value system"}
{"level":50,"time":"2024-10-17T11:59:30.558Z","msg":"sql.initialize was unsuccessful; returning early"}
{"level":30,"time":"2024-10-17T11:59:30.558Z","msg":"close event {\"readyForShutdown\":false,\"shouldQuit\":false}"}
{"level":30,"time":"2024-10-17T11:59:30.559Z","msg":"maybeRequestCloseConfirmation: Checking to see if close confirmation is needed"}
{"level":50,"time":"2024-10-17T12:00:21.145Z","msg":"onDatabaseError: Quitting application"}
{"level":30,"time":"2024-10-17T12:00:21.148Z","msg":"main window closed event"}
{"level":30,"time":"2024-10-17T12:00:21.149Z","msg":"quit event {\"hasEventBeenPrevented\":false,\"windowCount\":0,\"mainWindowExists\":false}"}
{"level":50,"time":"2024-10-17T12:00:21.153Z","msg":"Error occurred in handler for 'sql-channel:read': {\"name\":\"SqliteError\"}"}
{"level":50,"time":"2024-10-17T12:00:21.157Z","msg":"maybeRequestCloseConfirmation: Response never received; continuing with close."}
{"level":50,"time":"2024-10-17T12:00:21.159Z","msg":"MainSQL close, failed: SqliteError: file is not a database\n    at Database.pragma ([REDACTED]/node_modules/@signalapp/better-sqlite3/lib/methods/pragma.js:11:31)\n    at getUserVersion ([REDACTED]/ts/sql/util.js:132:13)\n    at migrateSchemaVersion ([REDACTED]/ts/sql/Server.js:406:54)\n    at openAndMigrateDatabase ([REDACTED]/ts/sql/Server.js:438:5)\n    at openAndSetUpSQLCipher ([REDACTED]/ts/sql/Server.js:460:14)\n    at initialize ([REDACTED]/ts/sql/Server.js:507:10)\n    at MessagePort.<anonymous> ([REDACTED]/ts/sql/mainWorker.js:69:41)\n    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:820:20)\n    at MessagePort.<anonymous> (node:internal/per_context/messageport:23:28)"}
```

If it's relevant at all, I have recently changed the following environment variable
`SIGNAL_PASSWORD_STORE=basic` -> `SIGNAL_PASSWORD_STORE=gnome-libsecret`
But I had since relaunched Signal and verified that it worked ok with the new setting.

For troubleshooting this, I tried reverting the change but I got a different (expected) Databse Error:
>Unable to access the database encryption key because the OS encryption keyring backend has changed from gnome_libsecret to basic_text. This can occur if the desktop environment changes, for example between GNOME and KDE.
>
>Please switch to the previous desktop environemnt or try to run signal with the command line flag --password-store="gnome-libsecret"

**Comment by XenonSup** [comment]
Yes, I set that specifically on the last version (7.28). After setting it, I had quit Signal once and restarted it, and everything worked correctly.
Then after updating to 7.29 the error happened.
I did see in the linked upstream issue a mention of the issue occurring only after multiple restarts of the app, so it's possible that I didn't get a chance to see that failure mode and it just so happened that a "second restart of the app" only happened when relaunching to use 7.29.

I think I'll give up on restoring my profile and test switching the password store back and forth to see if I can contribute helpful information.

**Comment by bermeitinger-b** [comment]
You can't reverse the change once you used the keyring (e.g. with `gnome-libsecret`).

It is a one-way ticket. This will also have an effect, if someone changes from Gnome to KDE.

**Comment by XenonSup** [comment]
Understood.
I meant that I would reset the app and try both settings.

I have done that and determined that this is not a 7.29 upgrade issue.
This was present in 7.28, and is still present in 7.29, and it is strictly tied to `gnome-libsecret`
I'll close this issue as I realize this is a duplicate of https://github.com/flathub/org.signal.Signal/issues/723 and is related to https://github.com/flathub/org.signal.Signal/issues/753 and https://github.com/flathub/org.signal.Signal/issues/754
Sorry for the trouble.
